Modules/Core/Common/src/itkDirectory.cxx

```diff
@@ -58,7 +58,7 @@ Directory::PrintSelf(std::ostream & os, Indent indent) const
 bool
 Directory::Load(const char * dir)
 {
-  return m_Internal->Load(dir);
+  return static_cast<bool>(m_Internal->Load(dir));
```

Comment:
It might be cleared to explicitly compare to the "success" status:

    return m_Internal->Load(dir) == itksys::Status::Success()

Reply:
I tried this first! But there is no `operator==`.